<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BazzarOne</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/Background/window.jpeg);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_home">
      <attribute name="icon">
       <iconset resource="background.qrc">
        <normaloff>:/new/prefix1/Background/Home.png</normaloff>:/new/prefix1/Background/Home.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Home</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>761</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>50</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>BAZZAR ONE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>711</width>
         <height>361</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>170</y>
         <width>321</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>30</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Account : Admin</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Logout">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>500</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);
</string>
       </property>
       <property name="text">
        <string>LOGOUT</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_Authority">
      <attribute name="icon">
       <iconset>
        <normaloff>Background/authority.jpg</normaloff>Background/authority.jpg</iconset>
      </attribute>
      <attribute name="title">
       <string>Authority</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>0</y>
         <width>491</width>
         <height>411</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>UPDATE TAX</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>471</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ebrima</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Add Tax Percentage to add in price of products:</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>170</y>
          <width>451</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ebrima</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_taxper">
           <property name="font">
            <font>
             <family>Ebrima</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_addtax">
           <property name="font">
            <font>
             <family>Ebrima</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);
</string>
           </property>
           <property name="text">
            <string>ADD</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>3</y>
         <width>601</width>
         <height>541</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="listView_Acc">
          <property name="font">
           <font>
            <family>Ebrima</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_Acc_Info">
            <property name="font">
             <font>
              <family>Ebrima</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);
</string>
            </property>
            <property name="text">
             <string>Show Account Info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_deleteAcc">
            <property name="font">
             <font>
              <family>Ebrima</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);
</string>
            </property>
            <property name="text">
             <string>Delete Account</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_requests">
      <attribute name="icon">
       <iconset resource="background.qrc">
        <normaloff>:/new/prefix1/Background/Requests.png</normaloff>:/new/prefix1/Background/Requests.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Requests</string>
      </attribute>
      <widget class="QLabel" name="label_pic">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>50</y>
         <width>481</width>
         <height>451</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_itemname">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>40</y>
         <width>451</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_price">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>40</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_catagory">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>160</y>
         <width>341</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_quantityold">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>140</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_dealername">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>370</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_demail">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>430</y>
         <width>591</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_remaining">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>510</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>520</y>
         <width>158</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_previous">
          <property name="font">
           <font>
            <family>Ebrima</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);
</string>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_next">
          <property name="font">
           <font>
            <family>Ebrima</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);
</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>520</y>
         <width>158</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_Reject">
          <property name="font">
           <font>
            <family>Ebrima</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);
</string>
          </property>
          <property name="text">
           <string>Reject</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Accept">
          <property name="font">
           <font>
            <family>Ebrima</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);
</string>
          </property>
          <property name="text">
           <string>Accept</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_quantitynew">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>200</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_Profit">
      <attribute name="icon">
       <iconset resource="background.qrc">
        <normaloff>:/new/prefix1/Background/profit.png</normaloff>:/new/prefix1/Background/profit.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Transactions</string>
      </attribute>
      <widget class="QTableView" name="tableView_profit">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>60</y>
         <width>1121</width>
         <height>441</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>9</y>
         <width>241</width>
         <height>49</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BAZZARONE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Profit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>1101</width>
         <height>49</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
